llama-index-core/llama_index/core/indices/struct_store/sql_query.py

```diff
@@ -351,10 +351,11 @@ def __init__(
 
         self._synthesize_response = synthesize_response
         self._verbose = verbose
+        self._streaming = kwargs["streaming"] if "streaming" in kwargs else False
         super().__init__(
             callback_manager=callback_manager
             or callback_manager_from_settings_or_context(Settings, service_context),
-            **kwargs,
+            # **kwargs,
```

Comment:
why are we commenting out **kwargs here?

Reply:
Right. So if we do what the other comment proposes then this should no longer be a problem then, right?

Reply:
yup, right